500_records/100_post_records.md

```diff
@@ -11,7 +11,7 @@ This endpoint is ratelimited at 3 successfully submitted records per 20 minutes
 
 <div class='info-yellow'>
 <b>Access Restrictions:</b><br>
-Unless you set `status` to `SUBMITTED` (or omit the field), access to this endpoint requires at least `LIST_HELPER` permissions.
+Unless you set `status` to `SUBMITTED` (or omit the field), access to this endpoint requires at least `LIST_HELPER` permissions. The `raw_footage` field is required unless the `status` field is _not_ `SUBMITTED`.
```

Comment:
huh, that's sloppy implementation on my part. Should "is required unless list helper permissions", just like the video field